ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -369,7 +369,14 @@ public void onResponse(Call call, Response response) throws IOException {
               // Otherwise send the data in one big chunk, in the format that JS requested.
               String responseString = "";
               if (responseType.equals("text")) {
-                responseString = responseBody.string();
+                try {
+                  responseString = responseBody.string();
```

Comment:
I think `response.request()` has that information. I will try to do that and amend the PR.